### DISCUSSION TITLE
Runtime Status Error

### DISCUSSION THREAD

**Discussion by adannup** [community/community#12428] [Copilot Conversations] [answered]
Hi,
I'm having this issue when Copilot initialize:

'configuration.icons' must be a array

Version: 1.7.5227

![Captura de Pantalla 2022-03-04 a la(s) 19 46 34](https://user-images.githubusercontent.com/6006486/156862733-4936c654-11e5-4299-8d94-08c9dbebc657.png)

**Comment by davecheney** [comment, marked as answer]
Thank you for your report. This has been fixed. Please upgrade your VSCode and Copilot extensions.

**Reply by ninetynin** [reply]
![image](https://user-images.githubusercontent.com/91336664/157176481-cb3ade19-31ec-45a6-9192-cfbfe2f62e03.png)
Yes the icon issue is fixed when updating but the copilot is not suggesting anymore so when opened the suggestion windows also its showing 0/10 and is struck in that screen im facing this issue from 2 days back iirc

**Reply by davecheney** [reply]
Please open a new issue. Please include the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.
